lib/s3routes/routes/routePOST.ts

```diff
@@ -58,6 +58,10 @@ export default function routePOST(
                     corsHeaders));
     }
 
+    if (objectKey === undefined && Object.keys(query).length === 0) {
+        return api.callApiMethod('objectPost', request, response, log, (err, resHeaders) => routesUtils.responseNoBody(err, resHeaders, response, 204, log));
```

Comment:
this crash if err is not one of our error with a code, example of error sent:
```
Error: Unexpected end of multipart data
    at /home/mickael/scality/cloudserver/node_modules/@fastify/busboy/deps/dicer/lib/Dicer.js:54:28
 ```

Producing this crash: 
```
RangeError [ERR_HTTP_INVALID_STATUS_CODE]: Invalid status code: undefined
    at new NodeError (node:internal/errors:387:5)
    at ServerResponse.writeHead (node:_http_server:314:11)
    at Object.errorXMLResponse [as errorResponse] (/home/mickael/scality/cloudserver/node_modules/arsenal/build/lib/s3routes/routesUtils.js:142:18)
    at Object.responseNoBody (/home/mickael/scality/cloudserver/node_modules/arsenal/build/lib/s3routes/routesUtils.js:432:35)
    at /home/mickael/scality/cloudserver/node_modules/arsenal/build/lib/s3routes/routes/routePOST.js:62:105
    at /home/mickael/scality/cloudserver/lib/api/api.js:317:24
    at /home/mickael/scality/cloudserver/node_modules/async/dist/async.js:421:16
    at next (/home/mickael/scality/cloudserver/node_modules/async/dist/async.js:5302:29)
    at /home/mickael/scality/cloudserver/node_modules/async/dist/async.js:906:16
    at Busboy.<anonymous> (/home/mickael/scality/cloudserver/lib/api/api.js:238:32)
```